src/Model/Nodeinfo.php

```diff
@@ -156,12 +156,16 @@ public static function getServices()
 		return $services;
 	}
 
-	public static function getOrganization($config)
+	public static function getOrganization()
 	{
-		$organization = ['name' => null, 'contact' => null, 'account' => null];
+		$organization = [
+			'name' => null,
+			'contact' => null,
+			'account' => null
+		];
 
-		if (!empty($config->get('config', 'admin_email'))) {
-			$adminList = explode(',', str_replace(' ', '', $config->get('config', 'admin_email')));
+		if (!empty(DI::config()->get('config', 'admin_email'))) {
+			$adminList = explode(',', str_replace(' ', '', DI::config()->get('config', 'admin_email')));
```

Comment:
Please keep the explicit dependency, `DI::config()` is a temporary crutch for the legacy static `Config` but it shouldn't replace an actual object.
```suggestion
		if (!empty($config()->get('config', 'admin_email'))) {
			$adminList = explode(',', str_replace(' ', '', $config()->get('config', 'admin_email')));
```